src/plays/randombackgroundgenerator/Random-background-generator.js

```diff
@@ -0,0 +1,63 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import { useEffect, useState } from 'react';
+import anime from 'animejs/lib/anime.es.js';
+import './styles.css';
+
+// WARNING: Do not change the entry componenet name
+function Randombackgroundgenerator(props) {
+  // Shabbir's Code Start below.
+  const [amount, setAmount] = useState(90);
+  const [circle, setCircle] = useState(true);
+  const [content, setContent] = useState(' ');
+  const blocks = [...Array(amount + 1).keys()].slice(1);
+
+  const animate = () => {
+    // using built in animejs function
+    anime({
+      targets: '.block',
+      translateX: () => anime.random(-900, 900),
+      translateY: () => anime.random(-500, 300),
+      scale: () => anime.random(1, 3)
+    });
+  };
+
+  const shapeChange = () => {
+    setCircle(!circle);
+  };
+
+  useEffect(() => {
+    animate();
+  });
+  // Shabbir's code end here
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+          {/* Shabbir's Code Starts Here */}
+          <div className="appContainer">
```

Comment:
Give this a unique name instead of `appContainer`.. and use that name as a prefix to all the classes in the .css file.